cpu/stm32/stmclk/stmclk_gx.c

```diff
@@ -155,9 +155,22 @@ void stmclk_init_sysclk(void)
     RCC->CR |= RCC_CR_PLLON;
     while (!(RCC->CR & RCC_CR_PLLRDY)) {}
 
+#if CLOCK_AHB > 80000000
```

Comment:
```suggestion
#if CLOCK_AHB > MHZ(80)
```

is a bit easier to read :)

Reply:
Addressed, as well as other new occurrences